pygeodiff/geodifflib.py

```diff
@@ -435,6 +435,19 @@ def read_changeset(self, changeset):
             raise GeoDiffLibError("Unable to open reader for: " + changeset)
         return ChangesetReader(self, reader_ptr)
 
+    def get_wkb_from_geometry(self, geometry):
+        func = self.lib.GEODIFF_createWkbFromGpkgHeader
+        func.argtypes = [ctypes.c_char_p, ctypes.POINTER(ctypes.c_size_t)]
+        func.restype = ctypes.POINTER(ctypes.c_char)
+        free_func = self.lib.GEODIFF_free
+        free_func.argtypes = [ctypes.c_char_p]
+        free_func.restype = None
+
+        size = ctypes.c_size_t(len(geometry))
+        ptr  = func(geometry, ctypes.byref(size))
+        result = ptr[:size.value]
```

Comment:
does this work, or you need to rather use https://docs.python.org/3/library/copy.html#copy.deepcopy ?

Reply:
well, seems test passes, so it should be ok.

Reply:
@wonder-sk do you know if deepcopy should be used here?